### PR TITLE
Dockerize2

### PR DESCRIPTION
**Pull request by nnur**
- Dockerized in dev and prod
- Goes through travis and is pushed live

Badabing!

### CODE REVIEW

File: .dockerignore [hunk 1]
```diff
@@ -1 +0,0 @@
-
```
Comment (dharness) [inline .dockerignore]: What is this change? If there is no change, please remove from PR. If this file does nothing, please remove file.

File: app/index.html [hunk 2]
```diff
@@ -4,12 +4,12 @@
...
     <head>
-        <title>Babelot</title>
+        <title>Bablot</title>
         <meta name="viewport" content="width=device-width, initial-scale=1">
         <meta charset="utf-8">
     </head>
     <base href="/">
     <body>
-        <babelot-app>Loading...</babelot-app>
+        <babelot-app>winning...</babelot-app>
```
Comment (dharness) [inline app/index.html]: Why is this needed?

File: deploy_key [hunk 3]
```diff
@@ -0,0 +1,51 @@
+-----BEGIN RSA PRIVATE KEY-----
```
Comment (dharness) [inline deploy_key]: This should not be on github, please remove and regenerate

File: deploy_key.pub [hunk 4]
```diff
@@ -0,0 +1 @@
+ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAACAQDWxbeajJUKnQOJYr4Y3NmmFfGorMH410B1IPU9LyAqLrJ6Y3UfwBinG3fhRNz2r7POVadCu13zY9IpDAEOlG9OChu74yRA0HLd8SKghu5z1i21qwfQ28Ko1bT85iGu3c+N1ffVio7Lym2yd8kt4p74AEhbwNdLwmPPx6QnhJYP1uivxgesfFX4gxN277vvog7gQB2uC6k1y+Q6IINZ6jC/5yTMiFk2t8H/o13Q1Ae9A/XIRRGUmpS6E6dDgHKoi+j8ta3OEWS3GyZewOH1NKeD06Ommt3nER/eLi7Jw7/FPQbhHMMRuYqSLgxb99nVCPqPrK0wxXWAfmFPJf5ZyEUzOPwteynBedW/C1q+BRCQGa8Xfcmz65C9YBQafeW4FkKhBNg4JS3+s4weVaIq9iyBJywBXZgYr786bicJlvY1kFwPFi19FFV8S/ETeBT+z1Nd/myCLGdxlHT+1Krqflou7AkwzyFAVs2Y4RRTFTV5ziF0dQkQfMCnrr2VAJCLHGQn1BKi2OXfNxw+mCONUZG2+GAg5uPIqG+ekeO//xuBEjbwGqH8fo0DhGopBL6GQ8J7C4sHlbjxVcjfNDAeb086BogTk+FQFWu66oA+oIgo/OEisG1Plpc35d1IJPbtyU02ZQxM7nXe8Y67CPoSqJ6RU64UEGsojjcKYxDQPMtjNQ== 23naila.n@gmail.com
```
Comment (dharness) [inline deploy_key.pub]: Same here

File: server.js [hunk 5]
```diff
@@ -1,5 +1,5 @@
 var express = require('express');
```
Comment (dharness) [inline server.js]: can this be 80 in prod, but 8000 in dev? Meaning, can you set it in docker-compose.prod.yml

Comment (nnur) [inline server.js]: done

File: deploy_key.pub (hunk 4 above)
Comment (nnur) [inline deploy_key.pub]: done

File: deploy_key (hunk 3 above)
Comment (nnur) [inline deploy_key]: done

File: app/index.html (hunk 2 above)
Comment (nnur) [inline app/index.html]: for testing travis

File: .dockerignore (hunk 1 above)
Comment (nnur) [inline .dockerignore]: removed

File: app/index.html (hunk 2 above)
Comment (dharness) [inline app/index.html]: Change it back to something sensible plz

**Comment by dharness** [comment]
LGTM
